src/components/pre-search/custom-query-preview.vue

```diff
@@ -87,6 +89,9 @@
     }
   })
   export default class CustomQueryPreview extends Vue {
```

Comment:
Right now this is sending the `origin=customer:results` parameter to the search.

![image](https://user-images.githubusercontent.com/68222542/193534170-190e8cce-c17a-4b24-bdbe-8cc64cb3b601.png)


 The location should be `no_query`. To do so, in `main.vue` you can wrap the `CustomQueryPreview` component with a `LocationProvider` component.